src/meson.build

```diff
@@ -0,0 +1,75 @@
+add_global_arguments([
+	'-I' + meson.current_build_dir(),
+	'-Db_sanitize=address'
```

Comment:
b_sanitize is a meson option, not a C option?

The -I flag doesn't seem to make much sense as the only compiled targets are in this very directory already, and the current build dir is already in the default list of `include_directories`. If not, you could always just add it to the root directory's

```meson

extra_includes = [
    include_directories('.'),
]
```

as 
```meson

extra_includes = [
    include_directories('.', 'src'),
]
```